UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -842,7 +842,7 @@ private void DequeueAndProcessServerShot()
 			if (shooter == PlayerManager.LocalPlayer)
 			{
 				//this is our gun so we need to update our predictions
-				FireCountDown += 1.0 / FireRate;
+				FireCountDown = FireDelay;
```

Comment:
I remember something about Autumn saying this should keep the +=

Reply:
Plus sign added.